pages/common/aws-s3-rm.md

```diff
@@ -0,0 +1,20 @@
+# aws s3 rm
+
+> Delete an S3 object.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/rm.html>.
+
+- Delete a single S3 object:
+
+`aws s3 rm s3://{{example_bucket_name}}/{{file_name}}.txt`
+
+- Perform a dry-run of a single S3 object deletion:
+
+`aws s3 rm s3://{{example_bucket_name}}/{{file_name}}.txt --dryrun`
+
+- Delete an object from an S3 access point:
+
+`aws s3 rm s3://arn:aws:s3:us-west-2:123456789012:{{access_point}}/{{my_access_point}}/{{my_key}}`
```

Comment:
I think this is the correct placeholders
```suggestion
`aws s3 rm s3://arn:aws:s3:{{region}}:123456789012:access_point/{{access_point_name}}/{{object_key}}`
```

Reply:
@navarroaxel I've updated it to look like this example from the AWS user guide [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-access-points.html)

```
arn:aws:s3:region:account-id:accesspoint/access-point-name/object/resource
```

Let me know if there shouldn't be curly braces around the `account_id` and `access_point` values!

Thanks for pointing this out